Readme.md

```diff
@@ -3,26 +3,40 @@
 [![Build Status](https://travis-ci.org/ddugovic/Stockfish.svg?branch=master)](https://travis-ci.org/ddugovic/Stockfish)
 [![Build Status](https://ci.appveyor.com/api/projects/status/github/ddugovic/Stockfish?svg=true)](https://ci.appveyor.com/project/ddugovic/Stockfish)
 
-Multi Variant Stockfish is a [Stockfish](https://github.com/official-stockfish/Stockfish) fork specialized to play chess and some chess [variants](https://en.lichess.org/variant):
-- [Crazyhouse](https://en.lichess.org/variant/crazyhouse)
-- [Atomic](https://en.lichess.org/variant/atomic)
-- [Horde](https://en.lichess.org/variant/horde)
-- [King of the Hill](https://en.lichess.org/variant/kingOfTheHill)
-- [Racing Kings](https://en.lichess.org/variant/racingKings)
-- [Giveaway](https://en.lichess.org/variant/antichess)
-- [Three-Check](https://en.lichess.org/variant/threeCheck)
-- [Chess960](https://en.lichess.org/variant/chess960) (played natively by Official Stockfish)
-
-The project has the goal to develop and to improve the playing strength of some chess variants not already played natively by Stockfish, once at month the maintainers will merge the Stockfish new patches to keep up to date also the playing strength for Chess and Chess960. If you are interested to play only Chess and/or Chess960, or to play Chess and/or Chess960 at maximum strength level possible, simply use Stockfish. Multi Variant Stockfish, though using the same code base for Chess and Chess960, is a bit slower.
-
-Multi Variant Stockfish is currently used in these [lichess.org](https://en.lichess.org/) projects: 
-- [lichess.org custom Stockfish](https://github.com/niklasf/Stockfish): as engine used to play against the users on lichess.org
-- [fishnet: distributed Stockfish analysis for lichess.org](https://github.com/niklasf/fishnet): as engine running on contributed remote servers for games analysis on lichess.org
+Multi Variant Stockfish is a [Stockfish](https://github.com/official-stockfish/Stockfish) fork specialized to play chess and some chess [variants](https://lichess.org/variant):
+- [Crazyhouse](https://lichess.org/variant/crazyhouse)
+- [Atomic](https://lichess.org/variant/atomic)
+- [Horde](https://lichess.org/variant/horde)
+- [King of the Hill](https://lichess.org/variant/kingOfTheHill)
+- [Racing Kings](https://lichess.org/variant/racingKings)
+- [Giveaway](https://lichess.org/variant/antichess)
+- [Three-Check](https://lichess.org/variant/threeCheck)
+- [Chess960](https://lichess.org/variant/chess960) (played natively by Official Stockfish)
+- Losers
+- Suicide
+- Loop
+
```

Comment:
Depending on whether we want a complete list or only a list of the most popular variants, we could add extinction, grid, and two kings chess.

Reply:
Let's add them all.